docs/KNOWN_BUGS

```diff
@@ -119,12 +119,6 @@ problems may have been fixed or changed somewhat since this was written.
 
 1. HTTP
 
-1.1 hyper memory-leaks
```

Comment:
Probably need to renumber the two entries below and also delete/renumber in the index.

Reply:
Nah, the script that converts that to show on the website handles gaps just fine. We prefer gaps than having to reorder every time we add/remove.

=> https://curl.se/docs/knownbugs.html